### PR TITLE
docs: bring back missing doc. Reorder table for consistency.

### PR DESCRIPTION
**Pull request by LProulx86**
GitHub Issue (If applicable): closes #
https://github.com/unoplatform/uno-private/issues/443

## PR Type

What kind of change does this PR introduce?

## What is the current behavior?

Some doc missing.

## What is the new behavior?

Reintroducing doc.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: doc/articles/includes/additional-windows-setup-inline.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+In order to run Skia+GTK heads, you will need to make sure to install the [GTK3 runtime](https://github.com/tschoonj/GTK-for-Windows-Runtime-Environment-Installer/releases).
```
Comment (agneszitte) [inline doc/articles/includes/additional-windows-setup-inline.md:1]: @jeromelaban not sure this is still needed for the Setup for Skia Desktop projects lately.
Is having a Windows tab still relevant here, please?
- https://github.com/unoplatform/uno/blob/1de939b926deccf26fec5372bfc96c943b24b02b/doc/articles/get-started-vs-2022.md?plain=1#L60
- https://github.com/unoplatform/uno/blob/1de939b926deccf26fec5372bfc96c943b24b02b/doc/articles/get-started-rider.md?plain=1#L49
- https://github.com/unoplatform/uno/blob/1de939b926deccf26fec5372bfc96c943b24b02b/doc/articles/common-issues-skia.md?plain=1#L18
- https://github.com/unoplatform/uno/blob/1de939b926deccf26fec5372bfc96c943b24b02b/doc/articles/features/using-skia-gtk.md?plain=1#L24

File: doc/articles/includes/additional-macos-setup-inline.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+In order to run Skia+GTK heads, you will need to make sure to install `gtk+3` package using the following command line, using [homebrew](https://brew.sh/):
```
Comment (agneszitte) [inline doc/articles/includes/additional-macos-setup-inline.md:1]: @jeromelaban not sure this is still needed for the Setup for Skia Desktop projects lately.
Is having a macOS tab still relevant here, please?
- https://github.com/unoplatform/uno/blob/1de939b926deccf26fec5372bfc96c943b24b02b/doc/articles/get-started-rider.md?plain=1#L57
- https://github.com/unoplatform/uno/blob/1de939b926deccf26fec5372bfc96c943b24b02b/doc/articles/common-issues-skia.md?plain=1#L26
- https://github.com/unoplatform/uno/blob/1de939b926deccf26fec5372bfc96c943b24b02b/doc/articles/features/using-skia-gtk.md?plain=1#L32

Comment (jeromelaban) [reply doc/articles/includes/additional-macos-setup-inline.md:1]: any gtk reference is not needed. If the section becomes empty, we can remove it.

Comment (agneszitte) [reply doc/articles/includes/additional-macos-setup-inline.md:1]: You can remove the macos tabs on these pages @LProulx86. 
Make sure to remove the tab part if there is only one tab left for some pages.

File: doc/articles/includes/additional-windows-setup-inline.md (hunk 1 above)
Comment (agneszitte) [reply doc/articles/includes/additional-windows-setup-inline.md:1]: You can remove the Windows tabs on these pages @LProulx86. 
Make sure to remove the tab part if there is only one tab left for some pages.

File: doc/articles/includes/additional-macos-setup-inline.md (hunk 2 above)
Comment (LProulx86) [reply doc/articles/includes/additional-macos-setup-inline.md:1]: @jeromelaban 

[uno/doc/articles/features/using-skia-gtk.md](https://github.com/unoplatform/uno/blob/1de939b926deccf26fec5372bfc96c943b24b02b/doc/articles/features/using-skia-gtk.md?plain=1#L32)

I guess we keep gtk reference in this article.

Comment (jeromelaban) [reply doc/articles/includes/additional-macos-setup-inline.md:1]: Yes, gtk for this article is required for legacy information.

**Review by agneszitte — COMMENTED** [review]
Need additional fixes @LProulx86 
For markdown linting validation:
image

**Review by agneszitte — COMMENTED** [review]
This commit will need to be adjusted with for conventional commit
image

**Comment by agneszitte** [comment]
@Mergifyio backport release/stable/5.2

**Review by agneszitte — APPROVED** [review]

**Pull request merged**